src/Data/Managers/Helpers/QueryManager/ContinuousWithEnv.test.tsx

```diff
@@ -6,6 +6,7 @@ import { StoreProvider } from "easy-peasy";
 import { Either, RemoteData } from "@/Core";
 import { QueryManagerResolverImpl, QueryResolverImpl } from "@/Data";
 import { getStoreInstance } from "@/Data/Store";
+import { drawerQuery } from "@/Slices/Notification/Core/Query";
```

Comment:
drawerQuery feels like the wrong name. I know this is out of scope of this specific task, but just wanted to voice the observation.

Reply:
I think it will be removed when we move notification queries to v2